app/lib/dashboard_queries.rb

```diff
@@ -96,7 +96,9 @@ def shared_student_fields(student)
       :first_name,
       :last_name,
       :grade,
-      :id
+      :id,
```

Comment:
@kevinrobinson I wanted to call this out since it also means sending that info to the tardies dashboard. My thinking is eventually we'll want to add these filters there as well, and it doesn't do any harm, but if we want to keep this compartmentalized then this can be done specifically for the data sent in the discipline api.